### ISSUE TITLE
TypeError: 'NoneType' object is not callable for basic example

### ISSUE THREAD

**Issue by SilverJan** [swimlane/pyattck#125]
**Describe the bug**

When executing the basic example from https://pyattck.readthedocs.io/en/latest/ics/icsattck.html, an Exception is thrown.

**To Reproduce**

```python
➜  ~ python3
Python 3.8.10 (default, Jun 22 2022, 20:18:18) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyattck import Attck
>>> attack_mitre = Attck()
>>> technique_list = attack_mitre.ics.techniques
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck/attck.py", line 289, in ics
    from .ics import ICSAttck
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck/ics.py", line 7, in <module>
    class ICSAttck(Base):
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck/ics.py", line 42, in ICSAttck
    __attck = MitreAttck(**Base.config.get_data("ics_attck_json"))
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck_data/attack.py", line 55, in __init__
    raise te
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck_data/attack.py", line 53, in __init__
    self.__attrs_init__(**kwargs)
  File "<attrs generated init pyattck_data.attack.MitreAttck>", line 14, in __attrs_init__
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck_data/attack.py", line 66, in __attrs_post_init__
    raise te
  File "/home/jan/.local/lib/python3.8/site-packages/pyattck_data/attack.py", line 62, in __attrs_post_init__
    data = TYPE_MAP.get(item['type'])(**item)
TypeError: 'NoneType' object is not callable
```

**Expected behavior**

No error

**Screenshots**

see logs above 

**Desktop (please complete the following information):**

Ubuntu 20.04, python3

**Additional context**

**Comment by kastellanos** [comment]
It seems like the past month was released a new STIX object. The campaigns https://attack.mitre.org/campaigns/ 

What I found is that is not yet implemented on the pyattck-data lib.
https://github.com/swimlane/pyattck-data/blob/e704a5c46d916ee3fb898b8c16902d8e7035fbe4/src/pyattck_data/attack.py#L27

**Comment by Derekt2** [comment]
is there a workaround?

**Comment by driesbuyck** [comment]
> is there a workaround?

A very dirty hack I put in just to get my scripts working for now is to comment out the 'raise re' on line 66 in the file site-packages/pyattck_data/attack.py and replace it with a print statement until the repo owner can put out a fix.
![image](https://user-images.githubusercontent.com/13899345/200802109-0071e0b4-1cbc-4857-8f2a-a7d01eaed073.png)

**Comment by adamcysec** [comment]
This issue is still present. Pyattck is currently unusable.

**Comment by SirSkizo** [comment]
I have opened [this](https://stackoverflow.com/questions/75012181/impossible-to-retrieve-data-form-pyattck-module) question on stack overflow.

The module is unusable due to this issue ATTOW

**Comment by MSAdministrator** [comment]
@SirSkizo thus will be resolved once this gets merged in https://github.com/swimlane/pyattck-data/pull/80

**Comment by SirSkizo** [comment]
Just in case anyone still troubleshooting with this issue:

They have already fixed this issue in future releases. You just need to update your package `pyattck-data` form the bugged version `2.5.2` to `2.6.1` (or any newer).

If you are using pip, just run this:

```bash
pip install --upgrade pyattck-data 
```

If you are using conda (inside your venv):
```bash
conda update pyattck-data
```

I think you can close this issue. Thanks you all !!

**Comment by MSAdministrator** [comment]
This is being resolved in #130 - just waiting on tests to pass and then will merge
